src/pages/reciters/index.tsx

```
@@ -47,27 +48,21 @@ const RecitersListPage = ({ reciters }) => {
-      props: {
-        chaptersData,
-        reciters: reciters || [],
-      },
-    };
-  } catch (e) {
-    return {
-      notFound: true,
-      revalidate: REVALIDATION_PERIOD_ON_ERROR_SECONDS,
-    };
+    const availableRecitersResponse = await getAvailableReciters(locale, []);
+    reciters = availableRecitersResponse.reciters;
```

ESLint

[error] Use object destructuring.

(prefer-destructuring)